docs/integration_resources/software/deadline.md

```diff
@@ -1,5 +1,63 @@
 # Deadline
 
-Shift integration in Deadline is on the works!
+Shift integration in Deadline is in the works. The Shift plugin for Deadline is currently only available on Windows and supports execution of Shift workflows in batch mode or within Maya.

```

Comment:
Given that we are already documenting Deadline, is it necessary to keep "Shift integration in Deadline is in the works?

```suggestion
Shift integration in Deadline is in the works. The Shift plugin for Deadline is currently only available on Windows and supports the execution of Shift workflows in batch mode or within Maya.

```

Reply:
Yes, we should mention that it is in the works, right now we have the plugin so we can send a Shift workflow to Deadline. But we have no way to send any other jobs to deadline from Shift for the moment